control-plane/plugin/src/resources/pool.rs

```diff
@@ -484,36 +547,39 @@ impl Cordoning for Pool {
     type UREQ = Option<UncordonReq>;
 
     async fn cordon(id: &Self::ID, resources: &Self::CREQ, output: &OutputFormat) -> PluginResult {
-        let resources = resources.clone().unwrap_or(CordonReq {
-            replicas: true,
-            snapshots: false,
-            restores: true,
-        });
-        let body = models::PoolCordonReq::new_all(
-            resources.replicas,
-            resources.snapshots,
-            resources.restores,
-        );
+        let body = models::PoolCordonReq::from(resources.clone().unwrap_or_default());
         match RestClient::client()
             .pools_api()
             .put_pool_cordon(id, body)
             .await
         {
-            Ok(node) => match output {
+            Ok(pool) => match output {
                 OutputFormat::Yaml | OutputFormat::Json => {
                     // Print json or yaml based on output format.
-                    utils::print_table(output, node.into_body());
+                    utils::print_table(output, pool.into_body());
                 }
                 OutputFormat::None => {
                     // In case the output format is not specified, show a success message.
-                    println!("Pool {id} cordoned successfully")
+                    let constraints = pool_cordon_resources(&pool.into_body());
+                    println!(
+                        "Pool {id} cordoned successfully. Current constraints: {constraints:?}"
+                    );
                 }
             },
             Err(source) => {
                 if source.error_body().map(|b| b.kind)
                     == Some(models::rest_json_error::Kind::AlreadyExists)
                 {
-                    println!("Pool {id} already cordoned");
+                    let pool = RestClient::client()
+                        .pools_api()
+                        .get_pool(id)
+                        .await
+                        .map_err(|source| Error::GetPoolError {
+                            id: id.to_string(),
+                            source,
+                        })?;
+                    let constraints = pool_cordon_resources(&pool.into_body());
+                    println!("Pool {id} already cordoned. Current constraints: {constraints:?}");
```

Comment:
`println!("Pool {id} is already cordoned. Current constraints: {constraints:?}");` ?